Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -442,7 +442,9 @@ def create_pull_request(cls, repository, args, branch_point, callback=None, unbl
             pr_issue = existing_pr._metadata['issue']
             labels = pr_issue.labels
             did_remove = False
-            for to_remove in cls.MERGE_LABELS + cls.UNSAFE_MERGE_LABELS + ([cls.BLOCKED_LABEL] if unblock else []) + [cls.SKIP_EWS_LABEL]:
+            labels_to_remove = cls.MERGE_LABELS + cls.UNSAFE_MERGE_LABELS + ([cls.BLOCKED_LABEL] if unblock else []) + ([cls.SKIP_EWS_LABEL] if not existing_pr.draft else [])
```

Comment:
I think we want to provide a `--no-ews`/`--ews` flag to `git-webkit` here too...I would expect the default to be `ews=True` for most PRs, but `ews=False` for draft PRs.